app/ForkedStar.tsx

```diff
@@ -0,0 +1,76 @@
+import octokit from "@/lib/octokit";
+import { env } from "@/env.mjs";
+import { GoRepoForked } from "react-icons/go";
+import { GoStar } from "react-icons/go";
+import { GoPeople } from "react-icons/go";
+import { getContributors } from "@/lib/api";
+
+const GITHUB_ORG: string = env.NEXT_PUBLIC_GITHUB_ORG;
+
+export default async function ForkedStar() {
+  const count = await getData().then((data) => {
+    return data.reduce(
+      (acc, ls) => {
+        if (ls.forks) {
+          acc.forkCount += ls.forks;
+        }
+        if (ls.stargazers_count) {
+          acc.starCount += ls.stargazers_count;
+        }
+        return acc;
+      },
+      { forkCount: 0, starCount: 0 },
+    );
+  });
+
+  const contributors = await getContributors();
```

Comment:
Since we just need the length, it'd be much more performance to use `getContributorsSlugs` since this just returns list of file names and does not do any analysis and other computes.
```suggestion
  const contributors = await getContributorsSlugs();
```